_includes/header.html

```diff
@@ -32,12 +32,18 @@
             </div>
           </form>
           <ul class="nav navbar-nav navbar-right">
-            <li{% if page.nav == "docs" %} class="active"{% endif %}>
-              <a href="{{ site.docs_site_url }}">Documentation</a>
-            </li>
-            <li{% if page.nav == "contribute" %} class="active"{% endif %}>
-              <a href="{{ site.main_site_url }}/contributing.html">Contribute</a>
-            </li>
+            <li{% if page.nav == "docs" %} class="active"{% endif %}>
+              <a href="{{ site.getting_started_category_url }}">Getting Started</a>
+            </li>
+            <li{% if page.nav == "docs" %} class="active"{% endif %}>
+              <a href="{{ site.docs_site_url }}">Using Bazel</a>
+            </li>
+            <li{% if page.nav == "docs" %} class="active"{% endif %}>
+              <a href="{{ site.extending_category_url }}">Extending Bazel</a>
+            </li>
+            <li{% if page.nav == "contribute" %} class="active"{% endif %}>
+              <a href="{{ site.main_site_url }}/contributing.html">Contributing</a>
+            </li>
```

Comment:
These look off on the desktop:

<img width="734" alt="Screen Shot 2021-03-05 at 12 36 59 PM" src="https://user-images.githubusercontent.com/347918/110067926-99eb1000-7daf-11eb-87b0-70b6a94a2bb5.png">

Reply:
Looks like some significant whitespace characters made it in. I cleaned it up, try applying the code below:
```suggestion
            <li{% if page.nav == "docs" %} class="active" {% endif %}>
                <a href="{{ site.getting_started_category_url }}">Getting Started</a>
            </li>
            <li{% if page.nav == "docs" %} class="active" {% endif %}>
                <a href="{{ site.docs_site_url }}">Using Bazel</a>
            </li>
            <li{% if page.nav == "docs" %} class="active" {% endif %}>
                <a href="{{ site.extending_category_url }}">Extending Bazel</a>
            </li>
            <li{% if page.nav == "contribute" %} class="active" {% endif %}>
                <a href="{{ site.main_site_url }}/contributing.html">Contributing</a>
            </li>
```

Reply:
<img width="621" alt="Screen Shot 2021-03-05 at 12 48 06 PM" src="https://user-images.githubusercontent.com/347918/110068624-0b778e00-7db1-11eb-8fee-d016f344d1a3.png">

Looks fine now.